src/core/engine.js

```diff
@@ -507,19 +862,19 @@ function engineFactory({ expression = '', position = 0, maths = {} } = {}) {
          * Inserts a term in the expression at the current position
          * @param {string} name - The name of the term to insert
          * @returns {calculator}
-         * @fires termerror if the term to add is invalid
-         * @fires termadd when the term has been added
+         * @fires error if the term to add is invalid
+         * @fires term when the term has been added
          */
-        useTerm(name) {
-            const prefixed = rePrefixedTerm.test(name);
+        term(name) {
```

Comment:
**thought**: the renaming done in the following section of the API (`term`, `termList`, `variable`, `command`) is not all that convincing. Personally I preferred the old methods beginning with verbs (even if `use*` was not an ideal prefix). Naming is hard! But, since this API won't be consumed publicly outside of the calculator component, you can leave them.

Reply:
Agreed, I wanted to give simpler names, but yes the new names are not actually verbs.
I've changed them to: `insertTerm`, `interTermList`, `insertVarible`, and `invoke`

Reply:
Looks better now!